keps/sig-auth/2718-20210511-client-exec-proxy/README.md

```diff
@@ -683,6 +683,9 @@ of this feature?**
   Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
   implementation difficulties, etc.).
 
+  - Number of proxy restarts
+  - Number of failed requests
```

Comment:
If you actually mean to add these metrics, please use the names so people know what to look for in Prometheus